tools/dissector/README

```diff
@@ -9,13 +9,17 @@ supported.
 * See also
```

Comment:
Also change the line "The OpenDDS DCPS dissector provides a Wireshark 1.2.x/1.3.x compatible" on the top of this file. The dissector is no longer compatible with Wireshark 1.x.

Reply:
The dissector does still support Wireshark 1 or at least that is the intention, it just drops sample payload dissection support if it is compiled against 1. I have tested it with the last 1.x release, 1.12, although testing with older versions might be necessary.

Reply:
Actually, I realized I don't know why I assumed payload dissection had to be dropped other than I had to rewrite most of code for it. I just removed the preprocessor condition and it compiled and loaded the of the pcaps I've been using. So the whole dissector should work in at least Wireshark 1.12. I would have to see about ones before that because I'm not familiar with the API changes. The only thing I know that doesn't compile with Wireshark 1.12 is the FT_CHAR field display type value.